src/ngraph/frontend/onnx_import/op/pad.cpp

```diff
@@ -18,19 +18,46 @@
 
 #include "default_opset.hpp"
 #include "exceptions.hpp"
+#include "ngraph/builder/split.hpp"
 #include "ngraph/coordinate_diff.hpp"
 #include "ngraph/op/constant.hpp"
+#include "ngraph/op/convert.hpp"
 #include "ngraph/op/pad.hpp"
 #include "ngraph/shape.hpp"
 #include "pad.hpp"
 #include "utils/convpool.hpp"
 
+using namespace std;
+
 namespace ngraph
 {
     namespace onnx_import
     {
         namespace op
         {
+            ngraph::op::PadMode setPadMode(string mode)
```

Comment:
Additionally I think we could refine this function name to reflect it's behavior more accurately. This function actually doesn't **set** pad mode for its parameter. It returns `PadMode` enumerated object accordingly to provided argument, therefore I'd suggest renaming it to `get_pad_mode`.

Reply:
Done